src/components/ScreenWrapper.js

```diff
@@ -43,13 +43,14 @@ const ScreenWrapper = props => (
                 paddingStyle.paddingBottom = paddingBottom;
             }
 
+            const wrapperStyles = _.union([
+                props.style,
+                styles.flex1,
+                paddingStyle,
+            ]);
```

Comment:
Does it work to put this directly into the styles key instead of making a variable for it?